discord-command.js

```diff
@@ -0,0 +1,22 @@
+/*
+    This class holds all the commands for Discord
+*/ 
+class DiscordCommands {
+    constructor() {             
+       
+    }
+    function setBotStatus(bot, status){
+        bot.user.setStatus(status);
+    }
+
+    function sendMessageToUser(bot, id, message){
+        bot.users.fetch(id)
+        .then((user) => {
+          user.send(message);
+         })
+        .catch(message => console.log((message)));

```

Comment:
you need to return the promise
`return bot.users.fetch(id)`
catch is just console log the message,  so it's useless.
so you could remove the catch

Reply:
by the way sendMessage to user is already implemented in `discordBot.js`